### ISSUE TITLE
"Not Verified" is what it keeps saying?

### ISSUE THREAD

**Issue by BrentCB** [vertcoin/electrum-vtc#33]
I bought VertCoins and transferred them with Shapeshifter and they are in my Wallet, I think? B/c it just says "Not Verified" and I can't figure out why?

How do I get that transaction to be "Verified"???

**Comment by pknight007** [comment]
Not Verified is what it says until you've caught up to the latest block header locally. Try leaving your Electrum-VTC open for a while longer.

**Comment by BrentCB** [comment]
Thanks for the response:) But it's been more then 12 hours and it still says "Not Verified"?

How do I get it Verified?

**Comment by BrentCB** [comment]
Here's what I just put in the notes...

Thanks for the response:) But it's been more then 12 hours and it still
says "Not Verified"?

How do I get it Verified?



Brent Cameron
Owner @ *CopyWritingControl*
https://www.linkedin.com/feed
https://www.Facebook.com/CopyWritingControl
https://medium.com/@CopyWritingStud

**Comment by BrentCB** [comment]
why did you close this issue? It is STILL NOT Verified? 

Fine I'll get my coins out of there, delete this wallet and tell everybody how crappy this wallet is!

**Comment by metalicjames** [comment]
Which OS are you on?

**Comment by BrentCB** [comment]
​I'm on macOS Sierra version 10.12.6​



Brent Cameron
Owner @ *CopyWritingControl*
https://www.linkedin.com/feed
https://www.Facebook.com/CopyWritingControl
https://medium.com/@CopyWritingStud

**Comment by metalicjames** [comment]
Download https://vtconline.org/blockchain_headers and replace the file in `~/.electrum-vtc` (Use "Go" from finder to get there). Restart Electrum-VTC.

**Comment by BrentCB** [comment]
Thanks James, but I'm sorry b/c I have no idea how to do what you're asking?

I downloaded the file you sent, but I can't open it on my mac?

I'm also totally lost as to what you mean by "replace the file in
~/.electrum-vtc (Use "Go" from finder to get there)".

How do I do what you're asking me to do?

Thanks,

Brent



Brent Cameron
Owner @ *CopyWritingControl*
https://www.linkedin.com/feed
https://www.Facebook.com/CopyWritingControl
https://medium.com/@CopyWritingStud

**Comment by BrentCB** [comment]
never mind, I got my VertCoins out and will delete the wallet



Brent Cameron
Owner @ *CopyWritingControl*
https://www.Facebook.com/CopyWritingControl
https://medium.com/@CopyWritingStud

**Comment by goodc0re** [comment]
*FIXED, read below

I also noticed that after a couple of days incoming payments show the status "Not verified". I am also using the OSX version. Now on 2.9.3.3  I notice that it won't sync past block 467711. 

*What I did to fix Electrum Vertcoin: 
I downloaded https://vtconline.org/blockchain_headers and replaced the file as metalicjames said. 

noob friendly tutorial:
To do this you 

0) close Electrum Vertcoin
1) open Finder in OSX, 
2) hold down the alt/option key and click on "Go" , "Go to folder ..." 
3) then enter  ~/.electrum-vtc
4) paste the downloaded file into this folder and replace it
5) open Electrum Vertcoin again and wait for a while

**Comment by deuce1307** [comment]
I am having the same exact problem discussed above.  "Not Verified."  I am using OSX, Electrum 2.9.3.3 and I have tried the suggestions you have made, at least as I can understand them.  I am not a programmer, I have minimal knowledge of any of this. I think step by step instructions, given to people who you should assume have almost no idea what you are talking about above would be highly beneficial.

**Comment by deuce1307** [comment]
Hey, 

Appreciate the update.  I am trying the suggested steps again, we'll see if it gets verified.  I do seem to be having trouble staying connected to a node/network, so that probably isn't helping.

**Comment by deuce1307** [comment]
It worked in like two seconds.  Thanks!!!

**Comment by metalicjames** [comment]
Looks like we have a solution for this issue in the comments above now. Closing.

**Comment by tamlin-mike** [comment]
@metalicjames I disagree. This is not solved. Downloading the headers file and manually replacing the local one is not a solution - it's merely a workaround. The core bug remains. I have myself been hit by it now, and a google search displays I'm not alone in the exact stop position - 467711.
Also see #58 I opened before finding this (IMHO in error) closed issue.

**Comment by ilius** [comment]
@metalicjames Perhaps these bugs would be solved if we manage to merge [electrum-ltc](https://github.com/pooler/electrum-ltc) (which is very up to date with [electrum](//github.com/spesmilo/electrum)). They have also switched to Python 3 and Qt 5.

I tried to merge electrum-ltc, and I think I have managed to resolved all conflicts excepts those in "lib/" directory, which I'm sure I can resolve, it's too much and risky, and I don't know much about the algorithms underlying of VTC nor LTC or BTC).
